templates/jenkins-slave-defaults.erb

```diff
@@ -80,9 +78,12 @@ if [ -n "$FSROOT" ]; then
   FSROOT_ARG="-fsroot '$FSROOT'"
 fi
 
-if [ -n "$TOOL_LOCATIONS" ]; then
-  TOOL_LOCATIONS_ARG="-toolLocations '$TOOL_LOCATIONS'"
-fi
+<% if @tool_locations %>
+  TOOL_LOCATIONS_ARG=""
+  <% @tool_locations.split.each do |tool_location| %>
+    TOOL_LOCATIONS_ARG='$TOOL_LOCATIONS_ARG --toolLocation <%= tool_location.split(':').first -%>=<%= tool_location.split(':').last -%>'
```

Comment:
What if the string is already in the `Python-2.7=/usr/bin/python2.7` format?  If there are any existing users of the `tool_location` param, wouldn't they have to be using that form?